Dockerfile

```diff
@@ -3,8 +3,11 @@ WORKDIR /app
 
 COPY ./setup.py /app/setup.py
 COPY ./README.md /app/README.md
-RUN pip install -e ".[dev]"
+COPY ./run.sh /app/run.sh
+COPY ./pytest.ini /app/pytest.ini
+RUN pip install -e ".[dev,test]"
 COPY ./main.py /app/main.py
+COPY ./test_main.py /app/test_main.py
```

Comment:
With all these `COPY`s, it's it better to put all the files in a `src/` directory or something?

Reply:
`COPY . /app` with appropriate `.dockerignore` ?

Reply:
Lemme try.